enginetest/queries/json_scripts.go

```diff
@@ -333,6 +333,35 @@ var JsonScripts = []ScriptTest{
 			},
 		},
 	},
+	{
+		Name: "JSON -> and ->> operator support",
+		SetUpScript: []string{
+			"create table t (pk int primary key, col1 JSON);",
+			`insert into t values (1, JSON_OBJECT('key1', 1, 'key2', '"abc"'));`,
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    `select col1->'$.key1' from t;`,
+				Expected: []sql.Row{{types.JSONDocument{Val: 1}}},
+			},
+			{
+				Query:    `select col1->>'$.key2' from t;`,
+				Expected: []sql.Row{{"abc"}},
+			},
+			{
+				Query:    `select * from t where col1->'$.key1' = 1;`,
+				Expected: []sql.Row{{1, types.JSONDocument{Val: map[string]interface{}{"key1": 1, "key2": "\"abc\""}}}},
```

Comment:
Not sure which one is preferred, but we have a `MustJSON` method that is used in other enginetests

Reply:
Ooh... that is a super handy util function. Thanks for pointing it out!